yb-voyager/cmd/exportSchema.go

```diff
@@ -122,3 +133,25 @@ func clearSchemaIsExported(exportDir string) {
 	flagFilePath := filepath.Join(exportDir+"metainfo", "flags", "exportSchemaDone")
 	os.Remove(flagFilePath)
 }
+
+func createMetaInfoJsonFile() {
```

Comment:
In the miginfo.json have the `MigInfo` type and two functions:
```
func LoadMigInfo(exportDir string) (*MigInfo, error)
func SaveMigInfo(miginfo *MigInfo) error // SaveMigInfo finds exportDir from miginfo.exportDir
```
Normal usage pattern of miginfo would be similar to:
```
miginfo, err := LoadMigInfo(exportDir)
if err != nil {
  ...
}
miginfo.SourceDBType = ...
miginfo.SourceDBName = ...
err = SaveMigInfo(miginfo)
if err != nil {
  ...
}
```